courses/databases_sql/intro.md

```diff
@@ -1,7 +1,7 @@
 # Relational Databases
 
 ### Prerequisites
-*   Complete [Linux course](/linux_basics/intro/)
+*   Complete [Linux course](https://linkedin.github.io/school-of-sre/linux_basics/intro/)
```

Comment:
we should be using relative links for internal documents. re: https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages

current one will need to change if we shifted to other domain. mkdocs supports linking internal docs via relative paths

Reply:
can you open an issue @sanketplus ?
We can audit and standardise to relative URLs

Reply:
ironically this was the only link in the entire document using relative url. literally every other link was an absolute url.

Reply:
no worries @kai3x5 might have gone unnoticed earlier. have opened #52